source/extensions/omni.isaac.lab/omni/isaac/lab/managers/reward_manager.py

```diff
@@ -126,98 +178,164 @@ def compute(self, dt: float) -> torch.Tensor:
             dt: The time-step interval of the environment.
 
         Returns:
-            The net reward signal of shape (num_envs,).
+            A dictionary containing the net reward signal of shape (num_envs,) for each group.
```

Comment:
We return a tensor when user doesn't set things up. We should throw proper deprecation warnings to the user so that they adapt.

Both the docstring here in the return and also in the class docs need to be fixed.